swagger_documentation/swagger.json

```diff
@@ -11377,6 +11377,22 @@
                 }
             }
         },
+        "PrivateLinkResponse": {
+            "x-tag": "common",
+            "type": "object",
+            "properties": {
+                "location": {
+                    "description": "Url for private link",
+                    "type": "string"
+                },
+                "token": {
+                    "description": "Token for private link",
+                    "type": "string",
+                    "format": "url",
```

Comment:
Same comment as in the API PR. `format` should be within `location`

Reply:
Thank you @tudoramariei